### PR TITLE
Edit user

### PR DESCRIPTION
**Pull request by ddzyanis**
Adding addres, last\first names and bith for user edit page

### CODE REVIEW

File: .byebug_history [hunk 1]
```diff
@@ -0,0 +1 @@
+exit
```
Comment (dmokreckiy) [inline .byebug_history]: add this to .gitignore

File: app/models/user.rb [hunk 2]
```diff
@@ -24,49 +26,7 @@ def self.from_omniauth(auth)
...
-  #     # if data = session["devise.linkedin_data"] && session["devise.linkedin_data"]["extra"]["raw_info"]
-  #     #   user.email = data["email"] if user.email.blank?
-  #     # end
-  #     # if data = session["devise.twitter_data"] && session["devise.twitter_data"]["extra"]["raw_info"]
-  #     #   user.email = data["email"] if user.email.blank?
-  #     # end
-  #   end
-  # end
+  def create_address
+    self.address = Address.new  if self.address.nil?
```
Comment (dmokreckiy) [inline app/models/user.rb]: `if self.address.nil?` - what do you think is wrong in here?

Comment (ddzyanis) [inline app/models/user.rb]: self?

Comment (dmokreckiy) [inline app/models/user.rb]: I would say `.nil?`

Comment (ddzyanis) [inline app/models/user.rb]: what is wrong?

Comment (dmokreckiy) [inline app/models/user.rb]: [First here](https://web.archive.org/web/20120911072105/http://raflabs.com/blogs/silence-is-foo/2010/08/12/nil-true-false-ruby)
[Then here](https://gist.github.com/jfarmer/2647362)

**Pull request merged**